osu.Game/Screens/Menu/ButtonSystem.cs

```diff
@@ -332,19 +329,19 @@ private void updateLogoState(MenuState lastState = MenuState.Initial)
                 case MenuState.Exit:
                 case MenuState.Initial:
                     logoTracking = false;
+                    allowOverlays.Value = OverlayActivation.Disabled;
```

Comment:
I'm pretty sure we don't want this `Disabled`. If anything, I'd uncomment `UserTriggered` and set that here so we aren't outright disabling for the time being (even if the user triggered support is not in yet).

Reply:
Also this is probably a use-case for hiding overlays *once* via a method call, but not actually disabling. Without the `All` set below in the delayed delegate (which is bound to cause weirdness).

Reply:
Regarding the first comment: It completely falls flat as I noticed that any logic for that in `ButtonSystem` is now unnecessary.
Regarding the second comment: Every time the `MenuState` changes the visibility of the overlays is reevaluated as seen [here](https://github.com/Aergwyn/osu/blob/707af0209732ef29de79d4417c2aab9892ff8f89/osu.Game/Screens/Menu/MainMenu.cs#L67).